test/visual/test_cases/date_extrema/options

```diff
@@ -0,0 +1,17 @@
+series: [
+  {
+    y: "y",
+    label: "y"
+  }
+],
+axes: {
+  x: {
+    key: "x",
+    type: "date",
+    min: new Date('2014-01-01'),
+    max: new Date('2016-01-01'),
+    ticksFormatter: function formatDate(date) {
```

Comment:
Thanks! I just copied what I use in my app, so it's great that there's a better way.